core/templates/pages/contributor-dashboard-page/modal-templates/question-suggestion-review-modal.controller.spec.ts

```diff
@@ -513,15 +518,21 @@ describe('Question Suggestion Review Modal Controller', function() {
     }));
 
     it('should fetch the rejection message', function() {
-      const responseDict = {
-        messages: [
-          { text: 'Question submitted.' },
-          { text: 'This is a rejection.' }
-        ]
-      };
+      var messages = [
+        { text: 'Question submitted.' },
+        { text: 'This is a rejection.' }
+      ];
+
+      let spyObj = spyOn(
```

Comment:
The variable assignment is not changing so prefer `const` here. Also let's be more specific with the name, e.g. `fetchMessagesAsyncSpy`.

Reply:
Done, will take care from next time for more specific name to be use.
Thanks.